webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/NodeInventory.scala

```diff
@@ -378,12 +392,16 @@ final object SoftwareUpdateKind {
  * #TODO: other informations ? A Map[String, String] of things ?
  */
 final case class SoftwareUpdate(
-    name   : String
-  , version: String
-  , arch   : String
-  , from   : String
-  , kind   : SoftwareUpdateKind
-  , source : Option[String]
+    name       : String
+  , version    : String
```

Comment:
shouldn't we have also a normalizedVersion so that we'll be able to compare versions ?

Reply:
I think we need to wait to know what such a field will have. Perhaps it will be several structured fields